coin-flip/index.html

```diff
@@ -1,17 +1,17 @@
 <!DOCTYPE html>
-<html>
+<html lang="en-US">
 <head>
 <title>Coin Flipper | My Digital Spot</title>
 <link rel="stylesheet" href="index.css">
 <script src="script.js"></script>
 </head>
-<center>
```

Comment:
As I mentioned in the CSS file, I'm using `text-align: center` instead of the `<center>` tag since `<center>` is deprecated (no longer used).